src/main/java/org/session5/homework/FibonacciSeriesGenerator.java

```diff
@@ -0,0 +1,27 @@
+package org.session5.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FibonacciSeriesGenerator {
+
+    private FibonacciSeriesGenerator() {
+        throw new IllegalStateException("Fibonacci Series Generator");
+    }
+
+    public static ArrayList<Integer> generateTheFibonacciSeries(int firstNNumbersToGenerate) {
+        List<Integer> fibonacciSeries = new ArrayList<>();
+        switch (firstNNumbersToGenerate) {
+            case 0 -> fibonacciSeries.add(0);
```

Comment:
The Fibonacci series starts as: 0, 1, 1, 2, 3, .... If someone wants the first number, it should return just 0. If someone wants the first two numbers, it should return 0, 1. So, the logic for case 0 and case 1 is incorrect. We can have instead:
```
case 0 -> {}
case 1 -> fibonacciSeries.add(0);
case 2 -> fibonacciSeries.addAll(Arrays.asList(0, 1));
```